state_machine.py

```diff
@@ -73,9 +75,62 @@ def update_user(email, password, reset_password_hash):
         return {'updated_user':updated_user}
     except Exception as e:
         raise Exception({'error':str(e.message)})
-    
-
 
+def get_user_by_user_id(user_id):
+    try:
+        users = db.session.query(User).filter(User.id == user_id)
+        if users.count() <= 0:
+            return None
+        if users.count() == 1:
+            for i in users:
+                return vars(i)
+        if users.count() > 1:
+            raise Exception({'error':'voilates the unique ability!'})
+    except Exception as e:
+        raise Exception({'error':str(e.message)})
+    
+def create_document(user_id):
+    try:
+        document = Document(
+            user_id = user_id
+        )
+        db.session.add(document)
+        db.session.commit()
+        return document.id
+    except Exception as e:
+        raise Exception({'error':str(e.message)})
 
+def get_document_by_document_id(id):
+    try:
+        documents = db.session.query(Document).filter(Document.id == id)
+        if documents.count() <= 0:
+            return None
+        if documents.count() == 1:
+            for i in documents:
+                return vars(i)
+        if documents.count() > 1:
+            raise Exception({'error':'voilates the unique ability!'})
+    except Exception as e:
+        raise Exception({'error':str(e.message)})
 
+def token_valid_check(auth_header):
```

Comment:
this function name says it checks validity of token; why is it returning the user_id then; 
either it should return nothing, or it should return the entire decoded payload (and not just the user_id)

Reply:
okay will send the payload then, anyways the payload just has the user_id only